code/ui/alivehud/c4/C4Arm.cs

```diff
@@ -60,17 +60,17 @@ public C4Arm() : base()
             Panel actionButtons = new Panel(contentPanel);
             actionButtons.AddClass("action-button-panel");
 
-            actionButtons.Add.TranslationButton("C4_UI_PICKUP", "button action-button", () =>
+            actionButtons.Add.TranslationButton(new Globalization.TranslationData("C4_UI_PICKUP"), null, "button action-button", () =>
             {
                 C4Entity.PickUp(Entity.NetworkIdent, Local.Pawn.NetworkIdent);
             });
 
-            actionButtons.Add.TranslationButton("C4_UI_DESTROY", "button action-button", () =>
+            actionButtons.Add.TranslationButton(new Globalization.TranslationData("C4_UI_DESTROY"), null, "button action-button", () =>
```

Comment:
Yeah, `TranslationData` is essentially the ability to pass in the key plus args in one centralized object.

Before with `TranslationButton` we had `3` parameters you'd have to potentially pass in to access the `args` field of a `translationkey`.

For example, let's say we wanted to create a button with a translation string of `Testing {value0} string {value1}`

It would look like...

`Add.TranslationButton("TESTING_STRING", null, onClick, value0, value1)`

The `value0` and `value1` look like they are related to the translation button as opposed to just being the args used with the key.

Whenever we have a new `translation` component we can simply just have the first parameter be the `TranslationData` instead of having to define the `args` parameter at end of every single object that has a translation field.